sellercenter-category/src/main/java/com/gfg/sellercenter/category/reader/HttpReader.java

```diff
@@ -0,0 +1,61 @@
+package com.gfg.sellercenter.category.reader;
+
+import lombok.AllArgsConstructor;
+import org.json.JSONArray;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * The HttpReader will do a request via HTTP
+ * @author Dennis Munchausen <dennis.munchausen@global-fashion-group.com>
+ * @version 1.0
+ * @since 1.0
+ */
+@AllArgsConstructor
+public class HttpReader {
+
+    private final String fqdnUrl;
+
+    /**
+     * This json  object represent the actual setting value only on a global level
+     * @param productIds Description name of the field in the database (global setting)
+     * @return JSONArray It retrieves a json object with main important values for a setting
+     * @throws Exception throws an exception when the endpoint is not reachable or no setting were found
+     */
+    public JSONArray getProductCategoriesByProductIds(List<Integer> productIds) throws Exception
+    {
+        String productIdsConverted = productIds.stream().map(String::valueOf)
```

Comment:
Please, also make it unique. During previous testing we found that in some requests with tens of thousands of product we are reaching all possible limits of URL length.

Reply:
you mean productIds?

Reply:
Yes. To make sure that clients will not have problem when they will try to get categories for set of products like "123,123,123,124,124,124,124,124"

Reply:
:+1: